<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DJI_Onboard_API_Test</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Commands</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="BtnActivation">
       <property name="text">
        <string>Activation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_nav_open_close">
       <property name="text">
        <string>Obtain Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Takeoff">
       <property name="text">
        <string>Takeoff</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Landing">
       <property name="text">
        <string>Landing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_GoHome">
       <property name="text">
        <string>Go Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_simpletask">
       <property name="text">
        <string>Simple Task</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="btn_open_serialport">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/icon/Images/err.png</normaloff>:/icon/Images/err.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="serial_comboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_update_com">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="baud_comboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>54600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>117</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>User Setting</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>APP id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AppID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>API level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="AppLevel">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AppKey">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>UAV Info</string>
    </property>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Activation status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Battery capacity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Control device:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>211</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Activation_Status">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Battery_Capacity">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Control_Device">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCtrl">
    <property name="title">
     <string>Ctrl</string>
    </property>
    <addaction name="actionCtrl"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuCtrl"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCtrl">
   <property name="text">
    <string>ctrl</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
